src/framework/node.ml

```diff
@@ -56,3 +56,7 @@ let find_fundec (node: t) =
   | Statement stmt -> Cilfacade.find_stmt_fundec stmt
   | Function fd -> fd
   | FunctionEntry fd -> fd
+
+let of_id s = 
+  let id = int_of_string s in
+  Statement { dummyStmt with sid = id }
```

Comment:
This needs to handle the cases where `s` begins with `fun` or `ret` as well. To find the `fundec` by its ID for those cases, something hacky like this might work:
```ocaml
Cilfacade.find_varinfo_fundec {dummyFunDec.svar with vid = id}
```

Reply:
I cleaned up the code a bit, but didn't test it, so make sure it still works.

Reply:
Made sure, still works. Thanks for the cleanup.